.github/workflows/db-migration-test.yml

```
@@ -0,0 +1,123 @@
+
+    steps:
+      - uses: actions/checkout@v3
+      
+      - name: JDK 설치
+        uses: actions/setup-java@v4
+        with:
+          java-version: 17
+          distribution: temurin
+
+      - name: Gradle 캐시 적용
+        uses: actions/cache@v3
```

actionlint (1.7.7)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)